doc/classes/ArrayMesh.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="ArrayMesh" inherits="Mesh" category="Core" version="3.2">
 	<brief_description>
+		[Mesh] type that provides utility for constructing Surface from arrays.
```

Comment:
Is there a specific reason to capitalize "Surface"? Also, I'm not a native speaker but shouldn't it be "constructing **a** surface"?

Reply:
I think it could go either way. I am using surface as if it is a proper noun because a "surface" is a special concept in Godot. Similar to the way that nodes or classes would be capitalized. However, if it is unclear to you, then it will be unclear for beginners. So I will fix.